src/menu/menu.scss

```diff
@@ -149,6 +150,32 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
       }
     }
   }
+
+  &-skeleton {
```

Comment:
I suppose this is a bit out of scope as it deals with the colors defined in skeleton.scss, but the colors for high contrast light don't appear to change from light. High contrast dark does, although I'm guessing that is just because the background is darker? None of the (base) colors pass AA or AAA on the backgrounds used.
Light:
![image](https://user-images.githubusercontent.com/37936169/162259451-41ea1ff0-f1ed-442e-8037-4032ca5eb363.png)
High Contrast Light:
![image](https://user-images.githubusercontent.com/37936169/162259472-3a23b37f-b4a7-4662-ad9b-bfe109bce0aa.png)
Dark:
![image](https://user-images.githubusercontent.com/37936169/162259491-64e38df8-491e-47d7-8b3a-fb5dc6517b67.png)
High Contrast Dark:
![image](https://user-images.githubusercontent.com/37936169/162259511-fef25803-72e7-4157-a0e6-684f728c5b97.png)

Reply:
Yikes, we will definitely need to look into this.

Reply:
Does it need to be AA or AAA on skeleton?

Reply:
I think it needs to be AA in default themes and AAA in high contrast themes. Maybe create a separate issue for now?